dynsampler.go

```diff
@@ -10,11 +10,23 @@ type Sampler interface {
 	// sampler.
 	Start() error
 
-	// GetSampleRate will return the sample rate to use for the string given. You
-	// should call it with whatever key you choose to use to partition traffic
-	// into different sample rates.
+	// Stop halts the sampler and any background goroutines
+	Stop() error
+
+	// GetSampleRate will return the sample rate to use for the given key
+	// string. You should call it with whatever key you choose to use to
+	// partition traffic into different sample rates. It assumes that you're
+	// calling it for a single span (not a trace), and simply calls
+	// GetSampleRateMulti with 1 for the second parameter.
 	GetSampleRate(string) int
 
+	// GetSampleRateMulti will return the sample rate to use for the given key
+	// string. You should call it with whatever key you choose to use to
+	// partition traffic into different sample rates. It assumes you're calling
+	// it for a trace. The second parameter is the number of events this trace
+	// contains.
+	GetSampleRateMulti(string, int) int
```

Comment:
Since we are assuming Span vs Trace with these functions, would it make sense to include something about Span/Trace in the function names?

Reply:
I considered and rejected that idea because the sampler doesn't really have the concept of either, and could theoretically sample anything. So I think I should probably update the comments instead to use "span" and "trace" as examples.